pkgs/tools/admin/awscli2/default.nix

```diff
@@ -56,8 +56,37 @@ let
     });
   };
 
+
+  inherit (lib) licenses maintainers optionalString;
+
+  inherit (py.pkgs)
```

Comment:
As in the other PR, I find this duplication unreasonable. It spreads the origin declaration and use too far apart, will be a pain to update given the number of entries, and isn't necessary at all, because these symbols are used in straightfoward ways. I'd just do `with py.pkgs; [ ... ]` in those places and call it a day.